modules/doc/content/python/testers/Exodiff.md

```diff
@@ -0,0 +1,41 @@
+# Exodiff
+
+`Exodiff` tests compare `Exodus` output(s) for the test to a reference in the specified
+`gold_dir` folder. Exodus output(s) contains both field variables, scalar variables and postprocessors,
+the latter two defined as global variables. This tester picks up any difference between
+all fields at any spatial location, at any time step included in the output(s).
+Complete documentation for the `Exodiff` utility may be found [here](https://gsjaardema.github.io/seacas-docs/sphinx/html/index.html#exodiff).
+
+## Options
+
+Test configuration options are added to the `tests` file.
+
+- `abs_tol`: sets an absolute tolerance, defaults to 1e-10. Both absolute and relative tolerances must
+  be met for a test to pass.
+
+- `rel_tol`: sets a relative tolerance. Defaults to 5.5e-6.
+
+- `exodiff_opts`: additional options passed to the `Exodiff` utility on the command line. Their list
+  may be found in its [documentation](https://gsjaardema.github.io/seacas-docs/sphinx/html/index.html#exodiff)
+
+- `custom_cmp`: use a custom comparison. The output of the comparison
+  must match the specified file.
+
+- `map`: Use geometrical mapping to match up elements. This is usually a good idea because it makes files comparable between runs with Serial and Parallel Mesh. Defaults to True.
```

Comment:
```suggestion
- `map`: use geometrical mapping to match up elements. This is usually a good idea because it makes files comparable between runs with Serial and Parallel Mesh. Defaults to True.
```